frontend_lib/test/component/CardPopupCreateContent.spec.js

```diff
@@ -6,10 +6,10 @@ require('../../src/component/CardPopup/CardPopupCreateContent.styl')
 
 describe('<CardPopupCreateContent />', () => {
   const props = {
-    onClose: () => {},
-    onValidate: () => {},
+    onClose: () => { return 1 },
```

Comment:
why do you need these int returns ?
If you really need it, I'm expecting to see a test that checks the value of the return